src/Razor/src/Microsoft.AspNetCore.Razor.VSCode.Extension/src/RazorDevModeHelpers.ts

```diff
@@ -30,7 +30,7 @@ export async function registerRazorDevModeHelpers(context: vscode.ExtensionConte
             __dirname, '..', '..', '..', '..', '..', 'artifacts', 'bin', 'Microsoft.AspNetCore.Razor.OmniSharpPlugin', config, 'net472', 'Microsoft.AspNetCore.Razor.OmniSharpPlugin.dll');
 
         if (!fs.existsSync(pluginPath)) {
-            vscode.window.showErrorMessage(`The Razor Language Server O# plugin has not yet been built - could not find ${pluginPath}`);
+            void vscode.window.showErrorMessage(`The Razor Language Server O# plugin has not yet been built - could not find ${pluginPath}`);
```

Comment:
Kinda funky. What's this rule fall under?

Reply:
Just a guess, but `showErrorMessage` [returns a `Thenable<string>` ](https://vscode-docs.readthedocs.io/en/stable/extensionAPI/vscode-api/?q=showInformationMessage&check_keywords=yes&area=default#window.showErrorMessage)so could this just be explicitly acknowledging discarding the return type.

Reply:
@TanayParikh is correct. The Thenable for messages returns with the result of the pop-up, but since we don't care in this instance the correct thing to do is acknowledge that we're dropping it on the floor.